zerver/views/streams.py

```diff
@@ -503,6 +503,15 @@ def stream_exists_backend(request, user_profile, stream_id, autosubscribe):
         return json_success(result) # results are ignored for HEAD requests
     return json_response(data=result, status=404)
 
+@has_request_variables
+def json_get_stream_ids(request, user_profile, stream=REQ()):
```

Comment:
Let's say `json_get_stream_id` singular for the view function and the URL, since we only get one id.